packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -5,5 +5,54 @@
 # CWD is agoric-sdk
 here=./upgrade-test-scripts/agoric-upgrade-11
 
-# run zoe thru "null upgrade"
-$here/zoe-upgrade/zoe-upgrade-driver.sh
+# Pre-steps:
+#  * fill Wallets
+#  * install bundles
+#  * create instance of prober contract and run expecting no atomicRearrange
+#
+# Action:
+#  * upgrade Zoe and ZCF
+#
+# Finish
+#  * create instance of prober contract and run expecting to see atomicRearrange
+
+echo XXXX fill wallet XXXXXX
+agd tx bank send validator $GOV1ADDR  12340000000${ATOM_DENOM} --from validator --chain-id agoriclocal --keyring-backend test --yes
+agops vaults open --wantMinted 10000 --giveCollateral 2000 > wantIST
+agops perf satisfaction  --executeOffer wantIST  --from gov1 --keyring-backend test
+
+
+# This test needs bundles for Zoe, ZCF, and the prober to be installed in this
+# directory, with names matching *bundle.json. The bundleIds for Zoe and ZCF
+# must be updated in zcf-upgrade-script.js, while the prober's bundleId goes in
+# run-prober.script. The bundles can be generated by running the test in
+# vats/test/bootstrapTests/test-zcf-upgrade.js. The Zoe and ZCF bundle files are
+# generated in ~/.agoric/cache, and their bundleIds are logged in the test. The
+# prober bundle is generated by uncommenting `fs.writeFile(...)` in that test.
+#  !!   THERE HAS TO BE A BETTER WAY   !!
+
+
+echo XXXX install bundles XXXXXX
+for f in $here/zoe-full-upgrade/*bundle.json; do
```

Comment:
I see this PR adds the bundles generated from the current git version. I can see why we might need to check in bundles from some _older_ git version, but for the current version, I think we can do:

```
yarn bundle-source --cache-json /tmp packages/vats/src/vat-zoe.js Zoe-upgrade
ZOE_HASH=`jq -r .endoZipBase64Sha512 /tmp/bundle-Zoe-upgrade.json`
yarn bundle-source --cache-json /tmp packages/zoe/src/contractFacet/vatRoot.js Zcf-upgrade
ZCF_HASH=`jq -r .endoZipBase64Sha512 /tmp/bundle-Zcf-upgrade.json`
```

and then assert that `$ZOE_HASH` matches the one from `zcf-upgrade-script.js`